CanonicalLtd/snappy-docs
# Add X-Commit-ID and X-Hostname headers

This is so we'll be able to tell which version of the codebase is running on which pod in production or staging simply by curling the domain.

This PR also add updates from run script v2.1.1: https://github.com/canonical-webteam/generator-canonical-webteam/pull/43. Feel free to review that too if you wish.

QA
--

``` bash
$ docker build --build-arg commit_id=`git rev-parse HEAD` --tag snappy .  # Build the docker image, with the current commit id
$ docker run --rm --name thowaway -d -ti -p 80:80 snappy  # Run the built image in the background
$ curl -sI localhost | grep X-  # Curl it to check you get the returned headers
X-Commit-ID: fca9b0fe6de6beeec64e70a20369cb38526e5e29
X-Hostname: 06f9b749bafd
$ docker kill throwaway  # Kill the background container
```
